Ticket - 34933 J-L-8 report is not on the standard menu and columns are cut off the report

sewa67 opened the pull request:
Please review this change and merge into development branch 
Files - jcrep/r-jcstdN.w

sewa67 committed:
Ticket - 34933 J-L-8 report is not on the standard menu and columns are cut off the report
Files - jcrep/r-jcstdN.w

brad-advantzware reviewed (changes requested):
Sewa - The change to Resources/StdMenu is missing here.  Has that already been done?

sewa67 committed:
Ticket - 34933 J-L-8 report is not on the standard menu and columns are cut off the report
Files - Resources/stdMenu/menu.cor,Resources/stdMenu/menu.fol,Resources/stdMenu/menu.lst

brad-advantzware reviewed (changes requested):
These changes to the menu files should be a single additional line.  These should all be rejected.  I will assign to Ron because we are changing the way that we define menus anyway.

Commit:
Delete menu.cor
